Dockerfile

```diff
@@ -1,17 +1,40 @@
-FROM python:3.7
+FROM ubuntu:eoan
 
-WORKDIR /tmp/
-RUN curl -L https://ladybug-tools-releases.nyc3.digitaloceanspaces.com/Radiance_5.5.2_Linux.zip --output radiance.zip \
-&& unzip -p radiance.zip | tar xz \
-&& mv ./radiance-Linux/usr/local/radiance/bin/* /usr/local/bin \
-&& mv ./radiance-Linux/usr/local/radiance/lib/* /usr/local/lib \
-&& rm -rf `ls`
+MAINTAINER Ladybug Tools info@ladybug.tools
+
+# Install core software deps
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+    curl \
+    ca-certificates \
+    python3.7 \
+    python3-pip \
+    unzip \
+    git \
+    && rm -rf /var/lib/apt/lists/*
 
-ENV RAYPATH=/usr/local/lib
+# Create non-root user
+RUN adduser ladybugbot --uid 1000 --disabled-password --gecos ""
+USER ladybugbot
+WORKDIR /home/ladybugbot
 
-WORKDIR /usr/app/
+# Install radiance
+ENV RAYPATH=/home/ladybugbot/lib
+ENV PATH="/home/ladybugbot/bin:${PATH}"
```

Comment:
Is this change because of potential permission issues? I liked the older version which is the common way that Radiance users are used to but I understand if we have to do this differently because of the permissions.

Reply:
The `RAYPATH` var is just set so that radiance knows where to find the lib folder. It does differ from the "out of the box" install style but means that there is less permissions manipulation (running `chmod` commands) with the `/usr/local` folders.

Line 23 however is used to add `/home/ladybugbot/bin` to the known executables path so that if you type `> rtrace` the shell knows to look in the `/home/ladybugbot/bin` folder for it. Installing user scoped binaries in the `/home/<user-name>/bin` is an accepted practice for linux environments.